
Determining the biological relevance and function of rare variants in NGS cancer data is a multistep process that can be very time and resource intensive. It is therefore incumbent upon us to strongly vet the variants gleaned from large exome and GWAS datasets before embarking onto in vitro studies.  As it currently stands, exome NGS data output from the CGR production laboratory passes through a standard bioinformatics pipeline before delivery to a DCEG investigator. The investigator reviews the data and selects several variants “of interest”, supposedly in a qualitative manner. In this way the investigator integrates knowledge of the disease, genetics and cohort characteristics, as well as the current literature.  The variants of interest are then presented to the functional genomics lab for further assessment of the variants and to determine feasibility for further study.  The software package HOTSPOT3D adds critical unbiased functionality to the functional genomics lab’s assessments of variants. The software maps variant calls onto 3-dimentional renderings of protein structure, allows for in silico protein-protein interaction mapping as well as protein-drug docking.  In this way, the functional genomics laboratory can help prioritize variants of critical interest from variants unlikely to yield results from further in vitro study.   
